Libs/RaptureAddins/ElasticSearch/src/test/java/rapture/elasticsearch/ReflexCallTest.java

```diff
@@ -0,0 +1,102 @@
+package rapture.elasticsearch;
```

Comment:
why is this unit test in ElasticSearch?  also it doesn't compile and is breaking the build